src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -157,4 +162,22 @@ private CaseDetails getRegistryDetails(CaseDetails caseDetails) {
         return caseDetails;
     }
 
+
+    @PostMapping(path = "/generate-deposit-receipt", consumes = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<WillLodgementCallbackResponse> generateDepositReceipt(@RequestBody WillLodgementCallbackRequest callbackRequest) {
+        Document document;
+        DocumentType template = WILL_LODGEMENT_DEPOSIT_RECEIPT;
+
+        Registry registry = registriesProperties.getRegistries().get("london");
```

Comment:
I am assuming that the location has to always be London, so you can't use:
registriesProperties.getRegistries().get(caseData.getRegistryLocation().toLowerCase())
Maybe worth adding "london" to the Constants file:
package uk.gov.hmcts.probate.model.Constants;